src/libs/NetworkConnection.ts

```diff
@@ -79,12 +79,12 @@ Onyx.connect({
         } else {
             // If we are no longer forcing offline fetch the NetInfo to set isOffline appropriately
             NetInfo.fetch().then((state) => {
-                const isInternetReachable = !!state.isInternetReachable;
+                const isInternetReachable = (state.isInternetReachable ?? false) === false;
                 setOfflineStatus(isInternetReachable);
                 Log.info(
                     `[NetworkStatus] The force-offline mode was turned off. Getting the device network status from NetInfo. Network state: ${JSON.stringify(
                         state,
-                    )}. Setting the offline status to: ${isInternetReachable}.`,
+                    )}. Setting the offline status to: ${!!isInternetReachable}.`,
```

Comment:
Why would we need !!?

Reply:
isInternetReachable is Already Boolean has value

Reply:
Revert this alone or explain 
cc: @pecanoro

Reply:
@Santhosh-Sellavel i updated